src/index.js

```diff
@@ -1,65 +1,50 @@
 import Evolv from '@evolv/javascript-sdk';
 
-function main(client, options) {
-	const context = {
-		web: {
-			url: window.location.href
-		},
-		user_attributes: options.user.attributes
-	}
 
-	client.initialize(
-		options.user.uid,
-		options.user.sid,
-		context
-	);
+const MAX_TIMEOUT_ATTEMPTS = 3;
+const MAX_TIMEOUT = 1000;
 
-	const maxTimeoutAttempts = 3
-	const maxTimeout = 1000
 
-	let timeoutAttempts = 0
+function main(client) {
+	let timeoutAttempts = 0;
 	let appliedClasses = [];
 	let unappliedFunctions = new Set();
 	let appliedFunctions = new Set();
 
 	function retrieveEvolvCssAsset() {
-		let cssAsset;
-	
-		const links = document.getElementsByTagName('link');
-
-		for (let i = 0; i < links.length; i++) {
-			const link = links[i];
-			if (link.rel === 'stylesheet' && link.href && link.href.indexOf('evolv.ai') >= 0 && link.href.indexOf('assets.css') >= 0) {
-				cssAsset = link;
-				break;
+		const styleSheets = document.styleSheets;
```

Comment:
I hesitated using this api bc MDN named it experimental. The browser support seemed to be extensive but it wasnt as extensive as `document.scripts`. https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/styleSheets